### PR TITLE
adding missing colors and events

### PR DESCRIPTION
**Pull request by Loottoo**
- Added missing colors and events.
- Fixed a bug where Color would autofill Color(Color(White)).
- Changed default number to 0 (making for example vector default Vector(0, 0, 0) rather than Vector(1, 1, 1). for loops also begin at index 0, rather than 1).

### CODE REVIEW

File: app/assets/stylesheets/elements/_codemirror.scss [hunk 1]
```diff
@@ -114,7 +114,9 @@
   .cm-completionIcon {
     padding-right: 1rem;
   }
-
+  .cm-completionIcon-event::after {
+    content: "∈";
+  }
```
Comment (CactusPuppy) [inline app/assets/stylesheets/elements/_codemirror.scss:119]: What's this for?

File: config/arrays/wiki/defaults.yml [hunk 2]
```diff
@@ -2,12 +2,12 @@ Compare: "True"
 Player: Event Player
 Primary Fire: Button(Primary Fire)
 Vector: Vector(0, 0, 0)
-Number: 1
+Number: 0
+Number (1): 1
```
Comment (CactusPuppy) [inline config/arrays/wiki/defaults.yml:6]: Kind of ugly, but it works.

File: app/assets/stylesheets/elements/_codemirror.scss (hunk 1 above)
Comment (Loottoo) [reply app/assets/stylesheets/elements/_codemirror.scss:119]: its the symbol shown in the editor, E for Event!
image

File: config/arrays/wiki/defaults.yml (hunk 2 above)
Comment (Loottoo) [reply config/arrays/wiki/defaults.yml:6]: yeah, some values really should default to 0, like the index for loops. others really shouldnt, like the incremented number or chase rate. not sure what else would be a better solution

**Pull request merged**